processing_scripts/Workflow7_ImageJ_Jython_Analyst.py

```diff
@@ -0,0 +1,236 @@
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+#------------------------------------------------------------------------------
+
+# This Jython script uses ImageJ to Subtract Background.
+# The purpose of the script is to be used in the Scripting Dialog
+# of Fiji.
+# Error handling is omitted to ease the reading of the script but this should be added
+# if used in production to make sure the services are closed
+# Information can be found at https://docs.openmicroscopy.org/omero/5.4.1/developers/Java.html
+
+import os
+from os import path
+
+from java.lang import Long
+from java.lang import String
+from java.lang import System
+from java.lang import Math
+from java.lang import Byte
+from java.util import ArrayList
+from java.util import Arrays
+from java.lang.reflect import Array
+from jarray import zeros, array
+import java
+
+# Omero Dependencies
+from omero.gateway import Gateway
+from omero.gateway import LoginCredentials
+from omero.gateway import SecurityContext
+from omero.gateway.facility import BrowseFacility
+from omero.gateway.facility import AdminFacility
+from omero.gateway.facility import DataManagerFacility
+from omero.gateway.facility import RawDataFacility
+from omero.gateway.facility import ROIFacility
+from omero.gateway.model import DatasetData
+from omero.log import Logger
+from omero.log import SimpleLogger
+
+from omero.model import ExperimenterGroupI
+from org.openmicroscopy.shoola.util.roi.io import ROIReader
+
+from ome.formats.importer import ImportConfig
+from ome.formats.importer import OMEROWrapper
+from ome.formats.importer import ImportLibrary
+from ome.formats.importer import ImportCandidates
+from ome.formats.importer.cli import ErrorHandler
+from ome.formats.importer.cli import LoggingImportMonitor
+import loci.common
+from loci.formats.in import DefaultMetadataOptions
+from loci.formats.in import MetadataLevel
+from loci.formats import FormatTools, ImageTools
+from loci.common import DataTools
+from loci.plugins.util import ImageProcessorReader
+
+from ij import IJ, ImagePlus, ImageStack, CompositeImage
+from ij.process import ByteProcessor, ShortProcessor
+from ij.process import ImageProcessor
+
+
+# Setup
+# =====
+
+# OMERO Server details
+HOST = "outreach.openmicroscopy.org"
+PORT = 4064
+group_id = -1
+#  parameters to edit
+image_id = 4528
+USERNAME = ""
+PASSWORD = ""
+
+# If you want to do analysis for someone else:
+# Mention their username here (you need to have light-admin previliges)
```

Comment:
``Mention`` -> state or define sounds better

Reply:
``(you need to have light-admin priviliges)`` -> (you need to have sufficient privileges) 

the full admin can execute too, so no need to be a light admin. Also, it is called restricted admin. Also, there was a typo ``priviliges``